powerfactory/powerfactory-converter/src/main/java/com/powsybl/powerfactory/converter/LineConverter.java

```diff
@@ -75,7 +76,9 @@ private void createFromElmLneFromElmTow(DataObject elmTow, DataObject elmLne) {
 
         NodeRef end1 = nodeRefs.get(0);
         NodeRef end2 = nodeRefs.get(1);
-
+        if (lineModel.isEmpty()) {
+            throw new PowsyblException("line Model cant be null to create a Line");
+        }
```

Comment:
Since `lineModel` can't be empty, Sonar raise an issue on this test.

I think the good way is to change the return type of `LineModel.createFromElmTow(...)`. There is no need for it to return an `Optional` and it can directly return the `LineModel`.